app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -18,9 +18,8 @@ module.exports = React.createClass
     # console.log 'Old marks were', @state.oldSetOfMarks
     newSetOfMarks = []
     # Automatically select new marks.
-    annotations = nextProps.classification?.annotations ? []
-    annotation = annotations[annotations.length - 1]
-    if annotation?
+    annotation = nextProps.annotation
```

Comment:
For every case I have tested it works.  Since the active task always appends to the end of the `classification.annotations` list, there should be no instance where the `annotation` passed on `props` does not match the last item in the classification.